### DISCUSSION TITLE
OBS HDMI input

### DISCUSSION THREAD

**Discussion by Lolmc0587** [Joshua-Riek/ubuntu-rockchip#704] [Q&A]
How i can use it with gstreamer on ubuntu 24.04 beta?

**Comment by Weicco** [comment]
I would also be interested to know if it is possible to make this HDMI input work with OBS. I don't think it's possible because it doesn't even work with the test utility above 5 fps. Unless you're a know-it-all nerd, of course. With the latest version, I couldn't get the HDMI input to work with any application at all. I want to believe that one day it will work. Joshua is doing a good job.

**Comment by radekolszak** [comment]
https://youtu.be/i_L0mGepOtI?si=JbU_QADfo0Yk8tC8
